### PR TITLE
Fix doom-emacs, again

### PR DESCRIPTION
**Pull request by thiagokokada**

### CODE REVIEW

File: checks.nix [hunk 1]
```diff
@@ -37,6 +37,7 @@ in
   init-example-el = self.outputs.package.${system} {
     doomPrivateDir = ./test/doom.d;
     dependencyOverrides = inputs;
+    emacsPackages = with pkgs; emacsPackagesFor emacsNativeComp;
```
Comment (thiagokokada) [inline checks.nix:40]: Using non NativeComp Emacs is broken right now, but I think this is an upstream issue.

Comment (thiagokokada) [reply checks.nix:40]: Opened an issue upstream: https://github.com/doomemacs/doomemacs/issues?q=is%3Aissue+native-comp-eln-load-path+is%3Aclosed

Comment (ckiee) [reply checks.nix:40]: Perhaps we should wait for upstream to fix it then. No point in shipping a broken version.

**Comment by thiagokokada** [comment]
Substitute PR: https://github.com/nix-community/nix-doom-emacs/pull/246

**Pull request closed without merging**